internal/kafka/test/integration/cluster_list_enterprise_test.go

```diff
@@ -84,4 +90,10 @@ func TestEnterpriseClustersList(t *testing.T) {
 	if resp2 != nil {
 		defer resp2.Body.Close()
 	}
+
+	checkMetricsError := common.WaitForMetricToBePresent(h, t, metrics.ClusterStatusCapacityAvailable, "3", api.StandardTypeSupport.String(), cluster.ClusterID, cluster.Region, cluster.CloudProvider)
+	g.Expect(checkMetricsError).NotTo(gomega.HaveOccurred())
+
+	checkMetricsError = common.WaitForMetricToBePresent(h, t, metrics.ClusterStatusCapacityMax, "3", api.StandardTypeSupport.String(), cluster.ClusterID, cluster.Region, cluster.CloudProvider)
+	g.Expect(checkMetricsError).NotTo(gomega.HaveOccurred())
```

Comment:
@pawelpaszki we could add this check here https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/46b36c569f59e89a45f0609bb2623dbe7b453ca6/internal/kafka/test/integration/cluster_metrics_test.go#L26

Reply:
Thanks. We can delete the check from here as well as it is duplicated.